Manifest.toml

```diff
@@ -0,0 +1,158 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
Did you mean to commit the manifest? That doesn't seem right. Drop it?

Reply:
yes, just temporarily, so we can run the tests with https://github.com/JuliaServices/ConcurrentUtilities.jl/pull/29 (before that's released)

will drop before merge